src/main/java/com/minecolonies/coremod/colony/ColonyList.java

```diff
@@ -117,7 +115,27 @@ public void add(T colony)
     }
 
     /**
+     * Get the Colony with the provided colony id.
+     *
+     * @param index colony id.
+     * @return The Colony associated with the provided id.
+     */
+    @Nullable
+    // no way to remove this, java does it too
+    @SuppressWarnings("unchecked")
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Suppressing warnings is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)